besu/src/main/java/org/hyperledger/besu/controller/IbftLegacyBesuControllerBuilder.java

```diff
@@ -58,7 +58,7 @@
   protected SubProtocolConfiguration createSubProtocolConfiguration(
       final EthProtocolManager ethProtocolManager) {
     return new SubProtocolConfiguration()
-        .withSubProtocol(Istanbul64Protocol.get(), ethProtocolManager);
+        .withSubProtocol(Istanbul99Protocol.get(), ethProtocolManager);
```

Comment:
Should Istanbul64 be supported as well as Istanbul99?

Reply:
I don't think I _could_ make istanbul64 work - something to do with Status message formatting.
Equally, the latest quorum docker is able to speak with with us with just this fix so, not sure I'm concerned about backward compatibility.